### PR TITLE
Bugfix/uod 941 fix ci docker build

### PR DESCRIPTION
**Pull request by romain-gilles-ultra**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -12,53 +10,59 @@ stages:
...
   - master
 
 # lint_code:
+#   extends:
+#     - .golang_based
 #   stage: test
 #   script:
 #     - make lint
 
 build:
```
Comment (olivier-norture-ultra) [inline .gitlab-ci.yml:63]:
```suggestion
build_and_publish:
```

Comment (olivier-norture-ultra) [reply .gitlab-ci.yml:63]: You do not plan to add a deploy step ?

**Review by olivier-norture-ultra — APPROVED** [review]
LGTM

File: .gitlab-ci.yml (hunk 1 above)
Comment (romain-gilles-ultra) [reply .gitlab-ci.yml:63]: No, it's in a different job but I rename this job `build-publish-docker`

**Review by olivier-norture-ultra — APPROVED** [review]

**Pull request merged**